### DISCUSSION TITLE
GitHub returns 503

### DISCUSSION THREAD

**Discussion by nbabanov** [community/community#22378] [Projects and Issues]
<p>Lately I started to receive a lot of 503 error happening at random.</p>
<pre><code class="lang-auto">Error: GET https://github.com/electron/electron/releases/download/v6.0.12/electron-v6.0.12-linux-x64.zip returned 503
</code></pre>

**Comment by Jason2866** [comment]
<p>We have this problem too.</p>
<pre><code class="lang-auto">Tool Manager: Installing https://github.com/tasmota/esptool/releases/download/v3.2/esptool-v3.2.zip
Error: Got the unrecognized status code '503' when downloaded https://github.com/tasmota/esptool/releases/download/v3.2/esptool-v3.2.zip
Error: Process completed with exit code 1.
</code></pre>

**Comment by jsoref** [comment]
<p>My guess is that if you logged the rest of the response you’d see that you hit an egress limit.</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/github/feedback/discussions/8535">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/github/feedback/discussions/8535" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/2119212/181070654-7fe5db45-4e77-4406-8f68-a78944f437fb.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/f/e/fe3cda7471624e2d47ecef185c90a487ebc34ca8_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/f/e/fe3cda7471624e2d47ecef185c90a487ebc34ca8_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/f/e/fe3cda7471624e2d47ecef185c90a487ebc34ca8.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/f/e/fe3cda7471624e2d47ecef185c90a487ebc34ca8_2_10x10.png"></div>

<h3><a href="https://github.com/github/feedback/discussions/8535" target="_blank" rel="noopener nofollow ugc">Release asset failing to download due to account egress limit · Discussion...</a></h3>

  <p>Hey all! We've seen this strange error when running terraform a few times today in our CI/CD environments, and it appears be saying that the release asset cannot be downloaded from github due t...</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>I can’t find any particular documentation about this limit. There are plenty of references to it as a thing and there are plenty of places where GitHub says that it reserves the right to enforce additional usage limits.</p>
<p>Since electron isn’t you, you could file an issue in their tracker – I don’t see one currently:</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/electron/electron/issues?q=is:issue%2Bis:open%2B503">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/electron/electron/issues?q=is:issue%2Bis:open%2B503" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/362;"><img src="https://user-images.githubusercontent.com/2119212/181070655-46e3bc93-3898-41e4-8dc9-ae5a7fa51336.png" class="thumbnail" width="690" height="362" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/c/8c19b0790c3ec6cebd14167bf62a792eda5a1d81_2_690x362.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/c/8c19b0790c3ec6cebd14167bf62a792eda5a1d81_2_1035x543.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/8/c/8c19b0790c3ec6cebd14167bf62a792eda5a1d81.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/c/8c19b0790c3ec6cebd14167bf62a792eda5a1d81_2_10x10.png"></div>

<h3><a href="https://github.com/electron/electron/issues?q=is:issue%2Bis:open%2B503" target="_blank" rel="noopener nofollow ugc">Issues · electron/electron</a></h3>

  <p>:electron: Build cross-platform desktop apps with JavaScript, HTML, and CSS - Issues · electron/electron</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>But, frankly, the fact that you are seeing this “a lot” tells me you’re doing something wrong.</p>
<p>You shouldn’t be downloading the same resource repeatedly. Save it somewhere! The act of downloading a resource is not free, it costs someone something.</p>

**Comment by nbabanov** [comment, marked as answer]
<p>I wrongly first posted here, before submitting a ticket to GitHub. They have resolved the issue and my pipelines are running without hiccups.</p>
<p><code>503</code> is a server error, so the issue was in GitHub’s code.</p>
<p>Note to someone having similar problems: File a support ticket from <a href="https://support.github.com" rel="noopener nofollow ugc">https://support.github.com</a></p>

**Comment by nbabanov** [comment]
<p>The error is not repo related - it is GitHub related. <code>503</code> means <code>Service Unavailable</code>.</p>
<p>Read more here: <a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/503" class="inline-onebox" rel="noopener nofollow ugc">503 Service Unavailable - HTTP | MDN</a></p>
